tests/unit/Imports/ProductImportTest.php

```diff
@@ -155,6 +157,209 @@ function (MockInterface $module) {
         $this->assertCount(1, $variableProduct->get_upsell_ids(), '1 upsell IDs should be set');
     }
 
+    public function dataProviderTestGetStockProperties()
```

Comment:
should also add backorderable checking to this unit test? that we have all covered. Or is there another test where it's covered

Reply:
Added backorders test